keras/applications/xception.py

```diff
@@ -144,7 +138,8 @@ def Xception(include_top=True, weights='imagenet',
         else:
             img_input = input_tensor
 
-    x = Conv2D(32, (3, 3), strides=(2, 2), use_bias=False, name='block1_conv1')(img_input)
+    x = ZeroPadding2D(padding=(1, 1))(img_input)
```

Comment:
I think there is a real performance concern with having a `ZeroPadding2D` layer at every block. Since there are a lot of blocks and this layer is known to be slow.

In addition, this model was trained with the architecture as previously defined for TF, so with these modifications there would be a slight mismatch between the pre-trained weights and the model.

I would rather have a slight result discrepancy across backends.